Adds a troubleshooting guide (#263)
bitnami/kube-prod-runtime: docs/troubleshooting.md

# Troubleshooting Guide

## Troubleshooting AKS cluster creation

### 1. AKS cluster creation fails

> If you notice the following error message while the cluster is being provisioned, it could indicate the Azure authentication token has expired.

Reviewer: ```suggestion
If you notice the following error message from `az aks create`, it could indicate the Azure authentication token has expired.
```

(to make it clear it is not from `kubeprod install aks`)

# Troubleshooting Guide

## Index

- [Troubleshooting AKS cluster creation](#troubleshooting-aks-cluster-creation)
    + [Service principal clientID not found](#service-principal-clientid-not-found)
- [Troubleshooting BKPR installation](#troubleshooting-bkpr-installation)
    - [Object with the same value for property exists](#object-with-the-same-value-for-property-exists)

## Troubleshooting AKS cluster creation

### Service principal clientID not found

If you notice the following error message from `az aks create`, it could indicate the Azure authentication token has expired.

>     ```

Reviewer 2: When I view this file in GitHub, the triple back ticks get rendered if they are indented. Perhaps you should remove the indentation?

Author: whoa.. i didn't even notice the rendering issue

Author: fixed!

    Operation failed with status: 'Bad Request'. Details: Service principal clientID: <REDACTED>
    not found in Active Directory tenant <REDACTED>, Please see https://aka.ms/acs-sp-help for more details.

>     ```

Reviewer 2: Ditto.
